catchpoint-mixin/targets.libsonnet

```diff
@@ -105,15 +106,235 @@ local utils = commonlib.utils {
-        'topk by (test_name) (2, max by (instance, test_name) (catchpoint_redirect_time{%(testNameSelector)s}))' % vars
+        'topk by (test_name, node_name) (2, max by (instance, test_name, node_name) (catchpoint_redirect_time{%(testNameSelector)s}))' % vars
       )
-      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(maxPanelLegend)),
 
     topErrorsByTestName:
       prometheusQuery.new(
         '${' + vars.datasources.prometheus.name + '}',
         'topk by (test_name) (2, max by (instance, test_name) (catchpoint_any_error{%(testNameSelector)s}))' % vars
       )
       + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+
+    // Web performance by test name dashboard
+    pageCompletionTime:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (node_name) (catchpoint_document_complete_time{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    DNSResolution:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (node_name) (catchpoint_dns_time{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    contentHandlingLoad:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (node_name) (catchpoint_content_load_time{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s - load' % utils.labelsToPanelLegend(testNameLabel)),
+
+    contentHandlingRender:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (node_name) (catchpoint_render_start_time{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s - render' % utils.labelsToPanelLegend(testNameLabel)),
+
+    clientProcessing:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (node_name) (catchpoint_client_time{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    additionalDelay:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (node_name) (catchpoint_redirect_time{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(testNameLabel)),
+
+    responseContentSize:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (node_name) (catchpoint_response_content_size{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('%s - content' % utils.labelsToPanelLegend(testNameLabel)),
+
+    responseHeaderSize:
```

Comment:
Response content size should have both the response content size and the response hender size. Additionally, it needs to display the node name in the legend too

Reply:
I think the response content size panel already has both portions.
<img width="695" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/a8b2c2b8-14e4-4c15-81a6-9fda4d6b652f">